README.md

```diff
@@ -205,7 +207,7 @@ c2patool sample/C.jpg --ingredient --output ./ingredient
 
 ### Adding a manifest to an asset file
 
-To add C2PA manifest data to a file, use the `--manifest` / `-m` option with a manifest JSON file as the option argument and the path to the asset file to be signed. Specify the output file as the argument to the `--output` / `-o` option. The output extension type must match the source. The tool will not convert between file types. For example:
+Use the `--manifest` / `-m` option to add the C2PA manifest store JSON file specified in the argument to the asset file to be signed. Specify the output file as the argument to the `--output` / `-o` option. The output extension type must match the source. The tool will not convert between file types. For example:
```

Comment:
I would not use manifest "store" here, the JSON file is only a single manifest. I'm used the term "manifest defintion file, or manifest definition JSON" at times. Manifest Store should refer to the entire store, either in the JUMBF or as a manifest store report in JSON.

Reply:
Reworded per your suggestion.  Thanks!